src/VisualStudio/Core/Def/SampleToolWindow/DocSymbol.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Imaging.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices
+{
+    using SymbolKind = LanguageServer.Protocol.SymbolKind;
+
+    internal class DocSymbol
```

Comment:
I would rename this type to `DocumentSymbolViewModel` to indicate that it is a helper type for the tree view instead of an actual document symbol